predicators/ground_truth_models/painting/painting_bridge_policy.ldl

```diff
@@ -0,0 +1,14 @@
+(define (policy)
+    (:rule OpenLid
+        :parameters (?lid - lid ?robot - robot)
+        :preconditions (and (GripperOpen ?robot) (PlaceFailed_arg0 ?robot) (not (IsOpen ?lid)))
+        :goals ()
+        :action (OpenLid ?lid ?robot)
+    )
+    (:rule PlaceOnTable
+        :parameters (?obj - obj ?robot - robot)
+        :preconditions (and (Holding ?obj) (HoldingTop ?obj) (IsClean ?obj) (IsDry ?obj) (OnTable ?obj) (PlaceFailed_arg0 ?robot) (not (GripperOpen ?robot)))
```

Comment:
we do have an operator, but that operator wouldn't be used by the planner because the planner thinks we can just put the held object into the box (without opening the lid)